Modules/Core/SpatialObjects/test/itkBoxSpatialObjectTest.cxx

```diff
@@ -71,6 +79,11 @@ itkBoxSpatialObjectTest(int argc, char * argv[])
   offset2[0] = 50.0;
   offset2[1] = 50.0;
   box2->SetPositionInObjectSpace(offset2);
+  for (unsigned int i = 0; i < Dimension; ++i)
+  {
+    ITK_TEST_SET_GET_VALUE(offset2[i], box2->GetPositionInObjectSpace()[i]);
+  }
```

Comment:
Although the above should work, it is maybe more a workaround: I am not sure if `offset2` was intended to be of type `BoxType::TransformType::OffsetType` or whether it should rather be a `BoxType::PointType` strictly speaking, since `itk::BoxSpatialObject::SetPositionInObjectSpace` expects the latter, but I guess the statement works due to some casting; however doing `ITK_TEST_SET_GET_VALUE(offset, box2->GetPositionInObjectSpace())` is not possible due to the type mismatch.

@aylward Can you please have a look? Thanks.

Reply:
Changed to `PointType` 266e1d3.